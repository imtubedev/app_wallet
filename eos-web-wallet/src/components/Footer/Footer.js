import * as React from "react";
import "./styles.module.scss";

const Footer = () => (
  <footer>
    <div className="columns u-pt2 is-variable is-3">
      <div className="column has-text-left-tablet has-text-centered-mobile">
        <p>Copyright 2017 | All Rights Reserved</p>
      </div>
      {/*<div className="column has-text-right-tablet has-text-centered-mobile">*/}
      {/*<a href="/privacy">Privacy Policy</a>*/}
      {/*<span> | </span>*/}
      {/*<a href="/terms">Terms of Service</a>*/}
      {/*</div>*/}
    </div>
  </footer>
);

export default Footer;
