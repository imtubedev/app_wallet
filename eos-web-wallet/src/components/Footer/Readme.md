A Footer example:

```js
<Footer />
```
