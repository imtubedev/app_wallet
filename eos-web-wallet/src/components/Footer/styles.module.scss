@import "../../styles/custom_variables.scss";

footer {
	border-top: 1px solid $grey-mid;
	margin: 3rem 3rem 0 3rem;
	padding-bottom: 1rem;
	font-size: 0.875rem;
	color: $grey-mid;
	p {
		color: $grey-mid;
	}
	a {
		color: $grey-mid;
		&:hover {
			color: $grey-mid;
		}	
	}
	@include mobile {
		margin: 2rem 2rem 1rem 2rem;
	}
}