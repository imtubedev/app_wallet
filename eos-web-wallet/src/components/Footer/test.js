import * as React from "react";
import renderer from 'react-test-renderer';
import { StaticRouter } from 'react-router';
import Footer from './';

describe('<Footer />', () => {
  it('renders without crashing', () => {
    const staticContext = {};
    const tree = renderer.create(
      <StaticRouter location="/" context={staticContext}>
        <Footer />
      </StaticRouter>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
})

