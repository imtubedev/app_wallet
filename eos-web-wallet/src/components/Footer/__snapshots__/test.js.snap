// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders without crashing 1`] = `
<footer>
  <div
    className="columns u-pt2 is-variable is-3"
  >
    <div
      className="column has-text-left-tablet has-text-centered-mobile"
    >
      <p>
        Copyright 2017 | All Rights Reserved
      </p>
    </div>
  </div>
</footer>
`;
